master/internal/provisioner/aws.go

```diff
@@ -169,37 +154,31 @@ func (c *awsCluster) list(ctx *actor.Context) ([]*Instance, error) {
 	res := c.newInstances(instances)
 	for _, inst := range res {
 		if inst.State == Unknown {
-			ctx.Log().Errorf("unknown instance state for instance %v", inst.ID)
+			ctx.Log().WithField("resource-pool", ctx.Self().Parent().Address().Local()).
```

Comment:
Is there not a way to add the `resource-pool` field to the logger earlier (so that `ctx.Log()` includes that field by default? Every log message in the provisioners needs to include the `resource-pool` field,